nixos/hosts/lab/rathole.vm.toml

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: Andrew Hayzen <ahayzen@gmail.com>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+# Note that this is the generated IP address of the test VM
+# they are alphabetical, backup, lab, vps
+#
+# TODO: can we make docker resolve "vps" to this IP?
+[client]
+remote_addr = "192.168.1.3:8333"
```

Comment:
can we teach docker where the "vps" host is?

Note `/etc/hosts` of lab is this, so we need `vps` to appear
```
vm-test-run-vps-test> 127.0.0.1 localhost
vm-test-run-vps-test> ::1 localhost
vm-test-run-vps-test> 127.0.0.2 lab
vm-test-run-vps-test> ::1 lab
vm-test-run-vps-test> vps actual.ahayzen.com bitwarden.ahayzen.com ahayzen.com yumekasaito.com
vm-test-run-vps-test> 192.168.1.1 backup
vm-test-run-vps-test> 192.168.1.2 lab
vm-test-run-vps-test> 192.168.1.3 vps
```

Reply:
Split into #132